content/docs/tech-insights/data-sources/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Data Sources
+publishedDate: '2022-11-15'
+description: Managing Data Sources.
+---
+
+## Introduction
+
+Data Sources periodically collect Facts about your services from third-party vendors or arbitrary APIs. The Facts collected by Data Sources are used for defining Checks. Roadie comes with a handful of built-in Data Sources with predefined Facts that you can use without any setup. You can also define your own Data Sources, which can use external APIs or files in a repository as a source of truth.
+
+To manage Data Sources, go to Tech Insights → Data Sources.
+
+### Using Built-in Data Sources
+
+By default, Roadie includes 5 Data Sources for common vendors such as Datadog, Snyk, and GitHub. All built-in Data Sources have pre-defined Facts you can use for defining your [Checks](../checks//index.md).
+
+To use built-in Data Sources, you don’t need to do any set up. Jump to the next section to learn more about Checks.
+
+### Defining your own Data Source
+
+Roadie Tech Insights lets you define arbitrary Data Sources. Typically, custom Data Sources rely on calling an API or reading a configuration file in a repository. When adding a Data Source, you’ll also specify which Facts are extracted from said source.
+
+To add a Data Source, click on the “Add Data Source” button in the Data Sources page.
+
+![Add Data Source](./data-sources-overview.png)
+
+To set up a Data Source, you’ll be prompted to define a Data Provider and define which fields will be extracted as Facts from the data. Let’s start overviewing the options available for setting up a Data Provider:
+
+![Add Data Provider](./data-provider-step.png)
+
+1 - You must specify a type for that new Data Source. The _HTTP_ type lets you connect to an external API to pull in data, while the _Component repository file_ type lets you extract data from a file path in the corresponding repository of a given Component in your Catalog. As an example, let’s use “_Component repository file”_.
```

Comment:
![Screenshot 2022-11-16 at 13 43 11](https://user-images.githubusercontent.com/562403/202196475-0d99b4c7-37d7-4af4-a348-a25c968db4ca.png)

There's a space missing between HTTP and type.

Reply:
akshually. It's the font that makes it look like that. There is a space

Reply:
Weird. Could we add two spaces then?